demo/dh-prod.yaml

```diff
@@ -0,0 +1,409 @@
+                                path: "/"
+                              route: { cluster: web }
+                            - match: # Web IDE lives in this path
+                                prefix: "/ide"
+                              route: { cluster: web }
+                            - match: # JS API lives in this path
+                                prefix: "/jsapi"
+                              route: { cluster: web }
+                            - match: # Notebook file storage at this path
+                                prefix: "/notebooks"
+                              route: { cluster: web }
+                            - match: # Any GRPC call is assumed to be forwarded to the real service
+                                prefix: "/"
+                              route:
+                                cluster: grpc-api
+                                max_stream_duration:
+                                  grpc_timeout_header_max: 0s
+                                timeout: 0s
+                  http_filters:
+                    - name: envoy.filters.http.health_check
+                      typed_config:
+                        "@type": type.googleapis.com/envoy.extensions.filters.http.health_check.v3.HealthCheck
+                        pass_through_mode: false
+                        headers:
+                          - name: ":path"
+                            exact_match: "/healthz"
+                          - name: "x-envoy-livenessprobe"
+                            exact_match: "healthz"
+                    - name: envoy.filters.http.grpc_web
+                    - name: envoy.filters.http.router
+            transport_socket:
+              name: envoy.transport_sockets.tls
+              typed_config:
+                "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.DownstreamTlsContext
+                common_tls_context:
+                  alpn_protocols: ["h2"]
+                  tls_certificates:
+                    - certificate_chain:
+                        filename: /etc/ssl/tls.crt
+                      private_key:
+                        filename: /etc/ssl/tls.key
+      clusters:
+        - name: grpc-api
+          connect_timeout: 10s
+          type: LOGICAL_DNS
+          lb_policy: ROUND_ROBIN
+          http2_protocol_options: {}
+          load_assignment:
+            cluster_name: grpc-proxy
+            endpoints:
+              - lb_endpoints:
+                  - endpoint:
+                      address:
+                        socket_address:
+                          # address: grpc-api # here we assume the name of the websocket proxy
+                          address: 127.0.0.1
+                          port_value: 8080
+        #      health_checks:
+        #        timeout: 1s
+        #        interval: 10s
+        #        unhealthy_threshold: 2
+        #        healthy_threshold: 2
+        #        grpc_health_check: { }
```

Comment:
Should the health checks be commented out?  If so, a comment would be hugely useful.